### PR TITLE
Revert "Common C integer types (#128)"

### PR DESCRIPTION
**Pull request by dcharkes**
This reverts commit 5e801e857662819d9bdb15df7dbb49d9ed60bb66.

By bumping the version number we can let `pub get` automatically upgrade from the 1.2.0 dev release. We had already retracted 1.2.0 stable, but we don't want to retract the dev release because ffigen dev releases depend on it.

Also, this cleans up the main branch of the repo to reflect the latest stable release of the package.

### CODE REVIEW

**Review by liamappelbe — COMMENTED** [review]
So iiuc, this change means that package:ffi supports SDK versions down to 2.12.0, but if the user wants the ABI specific types, they have to use version 2.17.0 or higher? Maybe we should mention that in the readme or something?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Changelog
 
-## 1.2.0
+## 1.2.1
+
+Revert added common C integer types as ABI-specific integers.
+Instead, these are available in Dart 2.17 instead.
```
[the remark this answers is not in the document]
Comment (dcharkes) [reply CHANGELOG.md]: done

**Comment by dcharkes** [comment]
> So iiuc, this change means that package:ffi supports SDK versions down to 2.12.0, but if the user wants the ABI specific types, they have to use version 2.17.0 or higher? Maybe we should mention that in the readme or something?

There are no ABI-specific types in this package at all. Unless they were using the dev release, they won't have ever seen them. Maybe adding it to the readme will add more confusion than clarification for people.

**Review by liamappelbe — APPROVED** [review]

**Review by jonasfj — APPROVED** [review]

**Comment by dcharkes** [comment]
> The base branch restricts merging to authorized users.

cc @mit-mit

**Pull request merged**